docs/manage/parts-and-remotes.md

```diff
@@ -12,7 +12,7 @@ This document explains how to establish secure connections between robots.
 
 ### Robot parts

```

Comment:
not sure why this is separate but I won't attempt to understand that now I guess

Reply:
Separate from what?

Reply:
Like why there's a part entry and a robot part entry

Reply:
There shouldn't be! Those are the same thing. Didn't realize you meant in the glossary because in this doc I called them robot parts to add context but part == robot part

Reply:
I think this "part" definition is good and "robot part" should be deleted entirely

Reply:
Oh wait I didn't mean here. I think there's a robot part glossary entry